packages/react-core/src/next/components/Modal/examples/ModalCustomHeader.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { Button, Title, TitleSizes } from '@patternfly/react-core';
+import { Modal, ModalBoxBody, ModalBoxFooter, ModalBoxHeader, ModalVariant } from '@patternfly/react-core/next';
+
+import WarningTriangleIcon from '@patternfly/react-icons/dist/esm/icons/warning-triangle-icon';
+import spacing from '@patternfly/react-styles/css/utilities/Spacing/spacing';
+
+export const ModalCustomHeaderFooter: React.FunctionComponent = () => {
+  const [isModalOpen, setIsModalOpen] = React.useState(false);
+
+  const handleModalToggle = (_event: KeyboardEvent | React.MouseEvent) => {
+    setIsModalOpen(!isModalOpen);
+  };
+
+  return (
+    <React.Fragment>
+      <Button variant="primary" onClick={handleModalToggle}>
+        Show custom header/footer modal
+      </Button>
+      <Modal
+        variant={ModalVariant.large}
+        isOpen={isModalOpen}
+        onClose={handleModalToggle}
+        aria-labelledby="modal-custom-header-label"
+        aria-describedby="modal-custom-header-description"
+      >
+        <ModalBoxHeader>
+          <Title id="modal-custom-header-label" headingLevel="h1" size={TitleSizes['2xl']}>
+            Custom header modal
+          </Title>
+          <p className={spacing.ptSm}>
```

Comment:
In the core examples, we avoid using the utility classes since they don't ship with PF. One way to space these two elements would be to use `<TextContent>` around them - that should be it, the title and paragraph don't need to use `<Text>` or anything.